tests/drivers/timer/nrf_grtc_timer/boards/nrf54h20dk_nrf54h20_cpuppr.overlay

```diff
@@ -0,0 +1,5 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+
+&grtc {
+	owned-channels = <5 6>;
```

Comment:
```suggestion
	owned-channels = <5>;
```
As you are passing to PPR only channel 5 (`owned channels` here should be consistent with `child-owned-channels` in `boards/nrf54h20dk_nrf54h20_cpuapp.overlay`

Reply:
That's fine, as one of the channels needs to be allocated by the system timer.